kolibri/plugins/pdf_viewer/assets/src/views/PdfRendererIndex.vue

```diff
@@ -127,7 +127,7 @@
   // polyfill necessary for recycle list
   import 'intersection-observer';
   import commonCoreStrings from 'kolibri.coreVue.mixins.commonCoreStrings';
-  import responsiveWindowMixin from 'kolibri.coreVue.mixins.responsiveWindowMixin';
+  import useKResponsiveWindow from 'kolibri-design-system/lib/useKResponsiveWindow';
```

Comment:
Could you please import the composable from `'kolibri.coreVue.composables.useKResponsiveWindow'`? This will ensure that if we ever change composables location in `kolibri-design-system`, we will then need to replace only one import in Kolibri in core vue rather than across many files